src/NuGet.Core/NuGet.Common/PublicAPI/net45/PublicAPI.Unshipped.txt

```diff
@@ -4,7 +4,6 @@ NuGet.Common.IPackLogMessage.LibraryId.get -> string
 NuGet.Common.IPackLogMessage.LibraryId.set -> void
 NuGet.Common.NuGetLogCode.NU1013 = 1013 -> NuGet.Common.NuGetLogCode
 NuGet.Common.NuGetLogCode.NU1301 = 1301 -> NuGet.Common.NuGetLogCode
-NuGet.Common.NuGetLogCode.NU1504 = 1504 -> NuGet.Common.NuGetLogCode
```

Comment:
Wasn't the warning the only reason why we added VersionOverride to LibraryDependency?
If we're removing the warning, then we can probably remove the VersionOverride there as well.

Reply:
I'm not sure, I added it to create the `DependencyGraphSpec` so that later on the central package versions can be applied.

https://github.com/NuGet/NuGet.Client/blob/71424bcf70087a60abc6ec6b86035bd572400464/src/NuGet.Core/NuGet.LibraryModel/LibraryDependency.cs#L142

This method is then called by all the places that construct the graph.  https://github.com/NuGet/NuGet.Client/search?q=ApplyCentralVersionInformation

Reply:
If Version is set at PackageSpec creation instead of VersionOverride, that should do it right?

Reply:
> Well I also have to check if the feature is disabled and log an error

Ah ok, so you still need it.

Yeah, if you feel like there's more opportunities to clean up, you can do it later.